students_json/mariany_queiroz.json

```
@@ -0,0 +1,70 @@
+{
+  "mariany_queiroz": "Mariany Queiroz",
```

acho q ficou confuso, desculpe! o nome do arquivo está correto agora :) mas a chave do json aqui deveria se manter assim: 
```suggestion
  "nome_completo": "Mariany Queiroz",
```
voce pode editar manualmente ou aceitar a sugestao por aqui mesmo :)